FogCreek/Glitch-Community
# Fallacious slice: Remove the global stylesheet

## Links
https://fallacious-slice.glitch.me/

## GIF/Screenshots:
It should look exactly the same as glitch.com, but maybe build a little faster

## Changes:
- Removed everything in the styles/ folder, and one of the webpack entry points with it
- Used shared components in place of a few things that still relied on global styles
- Moved the c3 styles into the team analytics code so it's at least located in the right place
- Moved the remaining global styles into the css modules globals

## How To Test:
- Try browsing around the site
- Check out team analytics
